cmd/workflow-controller/main.go

```diff
@@ -81,11 +81,7 @@ func NewRootCommand() *cobra.Command {
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 
-			wfController.UpdateConfig()
-
 			go wfController.Run(ctx, workflowWorkers, podWorkers)
-			go wfController.MetricsServer(ctx)
-			go wfController.TelemetryServer(ctx)
```

Comment:
These two servers were merged into one. This merged server is now started inside `wfController.Run`